openlibrary/data/dump.py

```diff
@@ -255,7 +254,7 @@ def _process_data(data):
             data['key'] = _process_key(data['key'])
 
         # convert date to ISO format
-        if 'type' in data and data['type'] == '/type/datetime':
+        if data.get('type', '') == '/type/datetime':
```

Comment:
Don't even need a default

```suggestion
        if data.get('type') == '/type/datetime':
```